src/entities/Publication.ts

```diff
@@ -0,0 +1,22 @@
+import { Field, ObjectType } from 'type-graphql';
+
+@ObjectType({ description: 'The Publications Model' })
+export default class Publication {
+  @Field()
+  slug: string;
+
+  @Field()
+  bio: string;
+
+  @Field()
+  feed: string;
```

Comment:
same thing about the Article <-> Publication reference, why is this a string?

Reply:
also if this is supposed to represent the articles belonging to this publication, wouldn't the name `articles` be more direct and clear? to me, a feed is a more generalistic collection of articles that are pulled together from multiple sources, so in that sense, it's awkward to say this "feed" belongs to this publication vs. these "articles" belong to this publication.

Reply:
agreed

Reply:
fixed, mostly was an issue with `rss-parser.ts` but I think it reads better now